mail_tracking/models/mail_tracking_email.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+import urlparse
+import time
+from datetime import datetime
+
+from openerp import models, api, fields, tools
+import openerp.addons.decimal_precision as dp
+
+_logger = logging.getLogger(__name__)
+
+
+class MailTrackingEmail(models.Model):
+    _name = "mail.tracking.email"
+    _order = 'time desc'
+    _rec_name = 'name'
+    _description = 'MailTracking email'
+
+    name = fields.Char(string="Subject", readonly=True)
+    timestamp = fields.Float(
+        string='UTC timestamp', readonly=True,
+        digits=dp.get_precision('MailTracking Timestamp'))
+    time = fields.Datetime(string="Time", readonly=True)
+    date = fields.Date(string="Date", readonly=True)
+    mail_message_id = fields.Many2one(
+        string="Message", comodel_name='mail.message', readonly=True)
+    mail_id = fields.Many2one(
+        string="Email", comodel_name='mail.mail', readonly=True)
+    partner_id = fields.Many2one(
+        string="Partner", comodel_name='res.partner', readonly=True)
+    recipient = fields.Char(string='Recipient email', readonly=True)
+    sender = fields.Char(string='Sender email', readonly=True)
+    state = fields.Selection([
+        ('error', 'Error'),
+        ('deferred', 'Deferred'),
+        ('sent', 'Sent'),
+        ('delivered', 'Delivered'),
+        ('opened', 'Opened'),
+        ('rejected', 'Rejected'),
+        ('spam', 'Spam'),
+        ('unsub', 'Unsubscribed'),
```

Comment:
Yes, not in this addon, but in other addon that inherits from it and implement this event. It is an standard event in mailing. There is an SMTP header for this also.